README.md

```diff
@@ -607,11 +607,12 @@ a {
 }
 ```
 
-### Scheme
+### Scheme - No autofix
```

Comment:
@imvetri 
Sorry, the **Scheme** is intended a single quick overview listing all possible values at once.
May I ask you to undo the this last change?

Reply:
Done!